### ISSUE TITLE
Make @javax.faces.bean.ManagedBean mean the same thing as @javax.inject.Named

### ISSUE THREAD

**Issue by glassfishrobot** [javaee/javaserverfaces-spec#1287]
The JSF 2.2 spec for the package javax.faces.bean states:

> The annotations in this package may be deprecated in a future version of this specification because they duplicate functionality provided by other specifications included in JavaEE. When possible, the corresponding annotations from the appropriate Java EE specification should be used in preference to these annotations.

Rather than deprecate, this issue suggests we re-purpose this package to just be equivalent to the equivalent items in CDI and JSR-330\.
#### Affected Versions
[2.0, 2.1, 2.2]

**Comment by glassfishrobot** [comment]
Reported by @edburns

**Comment by glassfishrobot** [comment]
Issue-Links:
is blocked by
[JAVASERVERFACES_SPEC_PUBLIC-1311](https://github.com/javaee/javaserverfaces-spec/issues/1311)
is related to
[JAVASERVERFACES_SPEC_PUBLIC-1316](https://github.com/javaee/javaserverfaces-spec/issues/1316)

**Comment by glassfishrobot** [comment]
@edburns said:
Set priority to baseline ahead of JSF 2.3 triage. Priorities will be assigned accurately after this exercise.

**Comment by glassfishrobot** [comment]
@manfredriem said:
Setting priority to Blocker

**Comment by glassfishrobot** [comment]
@manfredriem said:
Instead of doing this we have decided it is better to make tighter CDI support an opt-in for JSF 2.3\. So we are not going to retrofit to make it work in CDI.

**Comment by glassfishrobot** [comment]
Marked as **won't fix** on Thursday, January 15th 2015, 8:18:17 am

**Comment by glassfishrobot** [comment]
This issue was imported from java.net JIRA JAVASERVERFACES_SPEC_PUBLIC-1287
